docker/irods_client/tests/test_permissions.py

```diff
@@ -22,20 +22,18 @@ def test_permissions(session, item_name, request, tmpdir):
         upload(session, tmpdir/"bunny.rt.copy", ipath, overwrite=True)
     perm.set("own")
 
-    # TODO: Add more tests with other users.
-
-    # assert len(list(perm)) == 0
-    # ipath = IrodsPath(session, item.path)
-    # with pytest.raises(CAT_NO_ACCESS_PERMISSION):
-    #     download(session, ipath, tmpdir/"bunny.rt.copy", overwrite=True)
-    # perm.set("read", user=session.username, zone=session.zone)
-    # download(session, ipath, tmpdir/"bunny.rt.copy", overwrite=True)
-    # assert (tmpdir/"bunny.rt.copy").is_file
-    # with pytest.raises(ValueError):
-    #     upload(session, tmpdir/"bunny.rt.copy", ipath, overwrite=True)
-    # perm.set("write", user=session.username, zone=session.zone)
-    # # if item_name == "dataobject":
-    # upload(session, tmpdir/"bunny.rt.copy", ipath, overwrite=True)
-    # # else:
-    #     # upload(session, tmpdir/"bunny.rt.copy", ipath, overwrite=True)
-    # assert ipath.is_dataobject()
+@mark.parametrize("item_name", ["collection", "dataobject"])
```

Comment:
I guess we're not testing the write/read from the testuser at this point? That's fine with me, but perhaps something to be added later?

Reply:
Indeed!